### PR TITLE
Bump version and changelog for release 0.14.0

### PR DESCRIPTION
**Pull request by jalvz**
Required for https://github.com/elastic/package-storage/pull/688

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,19 @@ All notable changes to this project will be documented in this file.
...
+
+### Bugfixes
+
+### Added
+
+### Deprecated
+
+### Known Issues
+
+## [0.13.0](https://github.com/elastic/package-registry/compare/v0.13.0...v0.14.0)
```
Comment (mtojek) [inline CHANGELOG.md:19]: 0.14.0 ?

**Review by mtojek — COMMENTED** [review]
Would you mind filling in the PR description to explain why do you need to release the package registry?

**Comment by jalvz** [comment]
> Would you mind filling in the PR description to explain why do you need to release the package registry?

Sure

**Review by mtojek — APPROVED** [review]
LGTM. You can create a release tag.

**Pull request merged**